lib/drawer/xAxis.js

```diff
@@ -1,12 +1,13 @@
 import xAxis from '../xAxis';
 
-const drawAxis = (svg, xScale, configuration, orientation, y) => {
-    svg.append('g')
+const drawAxis = (container, xScale, configuration, orientation, y) => {
+    return container.append('g')
         .classed('x-axis', true)
         .classed(orientation, true)
-        .attr('transform', `translate(${configuration.margin.left}, ${y})`)
+        .attr('transform', `translate(0, ${y})`)
         .call(xAxis(xScale, configuration, orientation));
 };
 
-export const drawTopAxis = (svg, xScale, configuration, dimensions) => drawAxis(svg, xScale, configuration, 'top', configuration.margin.top - 40);
-export const drawBottomAxis = (svg, xScale, configuration, dimensions) => drawAxis(svg, xScale, configuration, 'bottom', +dimensions.height - 21);
+export const drawTopAxis = (container, xScale, configuration, dimensions) => drawAxis(container, xScale, configuration, 'top', 0);
+export const drawBottomAxis = (container, xScale, configuration, dimensions) => drawAxis(container, xScale, configuration, 'bottom', +dimensions.height - 21);
+export const boolOrReturnValue = (x, data) => typeof x === 'function' ? x(data) : x;
```

Comment:
That's a good practice. It is especially useful when you are working directly on CLI, for instance `cat`-ting files. Without end of line, you would get the prompt on the same line than your last file line.